src/main/app/src/app/formulieren/model/advies.ts

```diff
@@ -87,11 +87,12 @@ export class Advies extends AbstractFormulier {
         const dataElement = this.getDataElement(field);
         if (dataElement) {
             const zoekParameters = new EnkelvoudigInformatieObjectZoekParameters();
-            zoekParameters.UUIDs = dataElement.split(';');
-            return this.informatieObjectenService.listEnkelvoudigInformatieobjecten(zoekParameters);
-        } else {
-            return of([]);
+            if (JSON.parse(dataElement)?.selection) {
+                zoekParameters.UUIDs = JSON.parse(dataElement).selection?.split(';');
```

Comment:
Nu moet er 2 keer geparsed worden. Is het niet beter om het resultaat van die eerste parse actie te bewaren in een const genaams selection en als deze hier te gebruiken?

Reply:
Aangepast, ook voor goedkeuren.ts